src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -1116,6 +1116,7 @@ def dbapi(
         custom_schema: Optional[Union[str, StructType]] = None,
         predicates: Optional[List[str]] = None,
         session_init_statement: Optional[Union[str, List[str]]] = None,
+        external_access_integration: Optional[str] = None,
```

Comment:
to discuss: 

1. should we allow users to specify "package" and "import"
2. should we introduce another parameter to use udtf ingestion

Reply:
we can have a discussion tomorrow as I have some other findings